pages/data/explore.vue

```diff
@@ -45,114 +62,172 @@
 </template>
 
 <script>
-import WoudcBlurb from '~/components/WoudcBlurb'
-import WoudcSelect from '~/components/WoudcSelect'
-
-const now = new Date()
-
 export default {
-  components: {
-    'woudc-blurb': WoudcBlurb,
-    'woudc-select': WoudcSelect
-  },
   data() {
+    const now = (new Date()).getFullYear()
+
     return {
-      blurb: [
-        { text: this.$t('data.explore.blurb[0]') },
-        { newlines: 2 },
-        { text: this.$t('data.explore.blurb[1]') },
-        { newlines: 2 },
-        { text: this.$t('data.explore.blurb[2]') },
-        { link: { to: 'about-dataaccess', text: this.$t('data.explore.blurb[3]') } },
-        { text: this.$t('data.explore.blurb[4]') }
-      ],
-      selectedDataset: '',
-      selectedCountry: '',
-      selectedStation: '',
-      selectedInstrument: '',
+      selectedDataset: null,
+      selectedCountry: null,
+      selectedStation: null,
+      selectedInstrument: null,
       minYear: 1924,
-      maxYear: now.getFullYear(),
-      selectedYearRange: [1924, now.getFullYear()],
-      datasets: [
-        { text: 'All WOUDC Datasets', key: 'All' },
-        {
-          category: 'Total Column Ozone',
-          children: [
-            { text: 'Total Ozone - Daily Observations', key: 'TotalOzone' },
-            { text: 'Total Ozone - Hourly Observations', key: 'TotalOzoneObs' }
-          ]
-        },
+      maxYear: now,
+      selectedYearRange: [ 1924, now ],
```

Comment:
I think this might below in `computed`

Reply:
Working with `v-slider` is a bit weird, and it forces `selectedYearRange` to be in `data` rather than `computed`.

What do you think about setting a null initial value of `selectedYearRange` in `data`, and then giving it actual values in a `mounted` function that gets to use `maxYear` and `minYear`? Wanted to ask before committing.